src/bentoml/_internal/server/http_app.py

```diff
@@ -387,9 +387,6 @@ async def api_func(request: Request) -> Response:
                         )
                     else:
                         response = JSONResponse("", status_code=status)
-                except asyncio.CancelledError:
-                    # Special handling for Python 3.7 compatibility
-                    raise
```

Comment:
Hm, I don't remember but this feels like it's necessary even now, @frostming do you remember why this is 3.7 only?

Reply:
No, we can remove all `except CancelledError` now. Because on Python 3.7 CancelledError subclasses `Exception`, which will unexpected caught by the general `except Exception` block. It is changed to subclass `BaseException` on Python 3.8, so we don't need to bother now.

Reply:
There are some occurrences in other files, @aarnphm remove them as well.

Reply:
I see, yeah, we should remove them all @aarnphm.